catalyst_voices/packages/catalyst_voices_repositories/test/src/catalyst_data_gateway_repository/catalyst_data_gateway_repository_test.dart

```diff
@@ -1,82 +1,136 @@
 import 'package:catalyst_voices_models/catalyst_voices_models.dart';
 import 'package:catalyst_voices_repositories/src/catalyst_data_gateway_repository.dart';
+import 'package:catalyst_voices_services/generated/catalyst_gateway/cat_gateway_api.enums.swagger.dart' as enums;
 import 'package:catalyst_voices_services/generated/catalyst_gateway/cat_gateway_api.swagger.dart';
-import 'package:mockito/annotations.dart';
+import 'package:chopper/chopper.dart' as chopper;
+import 'package:http/http.dart' as http;
 import 'package:mockito/mockito.dart';
-import 'package:result_type/result_type.dart';
 import 'package:test/test.dart';
 
-import 'catalyst_data_gateway_repository_test.mocks.dart';
+// `@GenerateNiceMocks` from `mockito` can't be used here because
+// `chopper.Response` use the `base` modifier which "disallows 
+// implementations outside of its own library". For this reason
+// `@GenerateNiceMocks` doesn't work as intended and we opted to
+// mock the `CatGatewayApi` using `Fake`.
+class FakeCatGatewayApi<T> extends Fake implements CatGatewayApi {
+  final chopper.Response<T> response;
 
+  FakeCatGatewayApi(this.response);
+
+  @override
+  Future<chopper.Response<dynamic>> apiHealthStartedGet() async {
```

Comment:
How about using expressions here?

```
@override
Future<chopper.Response<dynamic>> apiHealthStartedGet() async => response;
```

For one-liners, especially if there are plenty of them helps to reduce the vertical space it takes